modules/nback.llvm/llvm.bal

```diff
@@ -114,19 +175,39 @@ public class Function {
 
 }
 
+# Corresponds to llvm::Module class
+public class Module {
+    private Function[] functions = [];
+    public function init() {
+    }
+
+    public function insertFunction(string name, FunctionType fnType) returns Function {
```

Comment:
Rename to addFunction to match LLVMAddFunction

Reply:
Fixed